src/policy.rs

```diff
@@ -80,10 +80,31 @@ impl PolicyAxis {
     }
 }
 
+#[derive(Clone, Serialize, Deserialize, Debug)]
+pub struct LegacyPolicy {
+    /// Last value taken by the attriute.
+    pub(crate) last_attribute_value: u32,
+    /// Maximum attribute value. Defines a maximum number of attribute
+    /// creations (revocations + addition).
+    pub max_attribute_creations: u32,
+    /// Policy axes: maps axes name to the list of associated attribute names
+    /// and a boolean defining whether or not this axis is hierarchical.
+    pub axes: HashMap<String, (Vec<String>, bool)>,
+    /// Maps an attribute to its values and its hybridization hint.
+    pub attributes: HashMap<Attribute, Vec<u32>>,
+}
+
+#[derive(Clone, Serialize, Deserialize, Debug, PartialEq, Eq, PartialOrd, Ord)]
+pub enum PolicyVersion {
+    V1,
```

Comment:
Should be V2, no?

Reply:
The previous version is called `LegacyPolicy` and has no version number. If I use `V2` for the current version, there would be no `V1`.